src/app/calculator/utilities/altitude-correction/altitude-correction-form/altitude-correction-form.component.html

```diff
@@ -0,0 +1,34 @@
+<form [formGroup]="form">
+    <div class="p-1">
+        <div class="form-group">
+            <label for="altitude">Altitude</label>
+            <div class="input-group">
+                <input class="form-control number-input-field" name="altitude" id="altitude" formControlName="altitude"
+                    type="number" (input)="save()" (change)="calculateBarometricPressure()"
+                    (focus)="focusField('altitude')" onfocus="this.select();">
+                <span *ngIf="settings.unitsOfMeasure != 'Metric'" class="input-group-addon units">ft</span>
+                <span *ngIf="settings.unitsOfMeasure == 'Metric'" class="input-group-addon units">m</span>
+                <!-- <span class="alert-danger pull-right small"
```

Comment:
We're requiring this field so validation for required should show. The rest/comments can be removed